### PR TITLE
Use the packages configuration data from DBus

### PR DESCRIPTION
**Pull request by poncovka**
Don't use the `%packages` section of the kickstart data. Use the DBus API instead.

**Depends on:** https://github.com/rhinstaller/anaconda/pull/3029

### CODE REVIEW

**Comment by poncovka** [comment]
Hi, I am not sure how to test in on Rawhide. I've tried to install `scap-security-guide`, but the installation ends up in a stack trace.

**Comment by matejak** [comment]
Thanks, this looks exciting. We have [https://github.com/OpenSCAP/oscap-anaconda-addon/blob/rhel8-branch/docs/manual/developer_guide.adoc](a test howto), but you probably got past it. I will also try this PR with a Fedora installation.

**Comment by matejak** [comment]
test this please

**Comment by matejak** [comment]
test this please

File: org_fedora_oscap/common.py [hunk 1]
```diff
@@ -617,3 +625,48 @@ def get_postinst_tailoring_path(data):
...
+
+def get_packages_data() -> PackagesConfigurationData:
+    """Get the DBus data with the packages configuration.
+
+    :return: a packages configuration
+    """
+    payload_proxy = get_payload_proxy()
+
+    if payload_proxy.Type != PAYLOAD_TYPE_DNF:
+        return PackagesConfigurationData()
```
Comment (poncovka) [inline org_fedora_oscap/common.py:652]: Just a note, this class should be replaced with `PackagesSelectionData` on Fedora Rawhide. The current version will work only on Fedora 34 and RHEL 9.

Comment (matejak) [reply org_fedora_oscap/common.py:652]: Thanks, time for a RHEL9 branch!

Comment (matejak) [reply org_fedora_oscap/common.py:652]: see #141

**Pull request merged**